cfgov/jinja2/v1/_includes/molecules/notification.html

```diff
@@ -0,0 +1,55 @@
+{# ==========================================================================
+
+   notification.render()
+
+   ==========================================================================
+
+   Description:
+
+   Builds notification markup when given:
+
+   options (optional):  An object used to customize the notification markup.
+
+   options.type:        The type of notification to display.
+                        Values may be: 'success', 'error', or 'warning'.
+
+   options.message:     Text to display within the notification markup.
+                        Default is 'Success.', 'Warning!', or 'Error!',
+                        depending on notification type.
+
+   options.explanation: Text to display as an explanation
+                        within the notification markup.
+
+   ========================================================================== #}
+{% macro render(options={}) -%}
+    {% set type  =  options.type if options.type in ['success', 'warning', 'error'] else 'success' %}
+    {% set icon_lookup  = {
+          'success' : 'approved',
+          'warning' : 'error',
+          'error'   : 'delete'
+        }
+    %}
+
+    {% if type == 'success' %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/eed72c63841678f6b7e388e0b4c5fd7955a8dd49